### PR TITLE
Adapt models

### PR DESCRIPTION
**Pull request by osswald**
- rename TextChoices: use "human readable" attributes for database values
- rename several classes: some consistency with the naming is helpful

### CODE REVIEW

File: train_management/locale/de_CH/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -196,111 +196,161 @@ msgid "length over bumper"
...
-#: models.py:146
+#: models.py:169
 msgid "date of birth"
 msgstr "Geburtsdatum"
 
-#: models.py:155
-msgid "Train"
-msgstr "Zug"
+#: models.py:178
+#, fuzzy
```
Comment (busykoala) [inline train_management/locale/de_CH/LC_MESSAGES/django.po:325]: Please remove

File: train_management/locale/de_CH/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -196,111 +196,161 @@ msgid "length over bumper"
...
+#| msgid "function_type"
+msgid "Function"
+msgstr "Funktionstyp"
+
+#: models.py:179
+#, fuzzy
```
Comment (busykoala) [inline train_management/locale/de_CH/LC_MESSAGES/django.po:331]: Please remove

File: train_management/models.py [hunk 3]
```diff
@@ -16,36 +16,36 @@ class Meta:
...
+    class Status(models.TextChoices):
+        AVAILABLE = "available", _("Available")
+        SERVICE = "servicing", _("In Servicing")
+        ASK = "ask", _("Ask")
 
     class CarriageType(models.TextChoices):
-        SEATS = "C", _("Seating Car")
-        GASTRO = "W", _("Waggon Restaurant")
-        CARGO = "H", _("Cargo")
+        SEATS = "seating", _("Seating Car")
```
Comment (haustre) [inline train_management/models.py:29]: SEAT = "seat"

**Review by busykoala — CHANGES_REQUESTED** [review]
Looks good, but please remove the fuzzy translations

File: train_management/models.py [hunk 4]
```diff
@@ -16,36 +16,36 @@ class Meta:
...
+        GASTRO = "restaurant", _("Waggon Restaurant")
```
Comment (haustre) [inline train_management/models.py:30]: same

File: train_management/models.py [hunk 5]
```diff
@@ -151,14 +174,18 @@ def __str__(self):
...
-        HINWIL = "H", _("Hinwil")
+        TRAIN = "train", _("Train")
+        BAUMA = "bauma", _("Bauma")
+        NEUTHAL = "neuthal", _("Neuthal")
+        BAERETSWIL = "baeretswil", _("Bäretswil")
+        HINWIL = "hinwil", _("Hinwil")
 
     label = models.CharField(_("label"), max_length=200)
-    label_short = models.CharField(_("label short"), max_length=10)
+    label_short = models.CharField(_("label short"), max_length=800)
```
Comment (haustre) [inline train_management/models.py:189]: No

**Review by haustre — CHANGES_REQUESTED** [review]

**Comment by osswald** [comment]
asdf

**Pull request closed without merging**